src/qubex/experiment/mixin/benchmarking_mixin.py

```
@@ -284,13 +284,13 @@ def rb_experiment_2q(
         self,
         *,
         target: str,
-        n_cliffords_range: ArrayLike = np.arange(0, 21, 2),
+        n_cliffords_range: ArrayLike = np.arange(0, 31, 3),
```

Ruff (0.8.2)

Do not perform function call `np.arange` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)